### DISCUSSION TITLE
Sometimes I cant type anything in the shells

### DISCUSSION THREAD

**Discussion by gerroon** [kingToolbox/WindTerm#563] [Q&A]
Hi

Does anyone have something similar to not being able to type in opened shells like cmd.exe, far.exe etc. I do not know what the issue but once in a while I am unable to type anything in the shells as in typing does not put characters on the terminals. I know I am not in the **Local** mode.

I am suspecting that I am invoking some key combination that enables it.

thanks

**Comment by kingToolbox** [comment]
Generally speaking, there are three situations that will cause characters to not be entered:

- The current mode is the `Local` mode, you need to press the `Ctrl+Enter` key to exit the mode.
- The input method is opening, you need to close the input method.
- You pressed `Ctrl+S` to pause `flow-control (XOFF)`, that means that the terminal will accept inputs but will not show the output of anything. To restart `flow-control`, simply press `Ctrl-Q (XON)` and you will see all your inputs compare on the screen.

If it is not for the reasons mentioned above, then it is likely to be the bug.
